test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -3021,24 +3000,23 @@ BOOST_AUTO_TEST_CASE(event_dynamic_array_storage)
 			event E(uint[]);
 			uint[] arr;
 			function createEvent(uint x) public {
-				arr.length = 3;
-				arr[0] = x;
-				arr[1] = x + 1;
-				arr[2] = x + 2;
+				arr.push(x);
+				arr.push(x + 1);
+				arr.push(x + 2);
 				emit E(arr);
 			}
 		}
 	)";
-	ALSO_VIA_YUL(
-		compileAndRun(sourceCode);
-		u256 x(42);
-		callContractFunction("createEvent(uint256)", x);
-		BOOST_REQUIRE_EQUAL(numLogs(), 1);
-		BOOST_CHECK_EQUAL(logAddress(0), m_contractAddress);
-		BOOST_CHECK(logData(0) == encodeArgs(0x20, 3, x, x + 1, x + 2));
-		BOOST_REQUIRE_EQUAL(numLogTopics(0), 1);
-		BOOST_CHECK_EQUAL(logTopic(0, 0), dev::keccak256(string("E(uint256[])")));
-	)
+	/// TODO enable again after push(..) was implemented.
```

Comment:
Is there an issue for this already? It would maybe be a good idea to open one before this gets merged otherwise we might forget to re-enable it.

Reply:
Good catch. This doesn't need to be deactivated any more, since `push()` via yul is available now.

Reply:
There are some more test cases which I re-enabled already.

Reply:
@leonardoalt The remaining ones are tracked with https://github.com/ethereum/solidity/issues/7749.